### ISSUE TITLE
Discussions & Questions

### ISSUE THREAD

**Issue by GHNewbiee** [JeffHoogland/moksha#149]
Sorry, but all community contact means you provide require a new registration/account. Github `Discussions` is a good way for all that means to get avoided. I hope you activate it soon.

Please, let me know if `Moksha Desktop` provides:

- multiple task/tool bars (in all themes I see only one at the bottom)
- menu editor (new app groups, new app items, and transfer of items between groups, the minimum required) (It is not clear in the "The Menu System" description)
- hotspot connection creation (anonymous or registered, and quick like KDE Plasma)

*Tia*

**Comment by rbtylee** [comment]
Yes, you can have multiple shelves (what you are calling " task/toolbars").

Bodhi is a minimalist distro, so there is no default application to provide any kind of menu editing beyond adding a new desktop file or editing an existing one. You are free to install such an application or to directly edit the menu file itself, or create or install another menu file. See /etc/xdg/menus as well as look up the freedesktop menu specifications.

I am unsure on hotspot creation. If network manager does not provide it out of the box then you most likely need to install some stuff. Keep in mind any applications libraries found in Ubuntu's repository can be installed on Bodhi. And all such applications should work UNLESS they are specific to a certain Desktop Environment.

**Comment by GHNewbiee** [comment]
> ... (what you are calling " task/toolbars").

In the following screenshot you can see 3 task/toolbars; one at the top, one at the bottom and another one on the right.

![Screenshot from 2021-09-28 10-58-21](https://user-images.githubusercontent.com/26875662/135057358-28c38965-052d-450b-b54b-2494b7f5dd1f.png)
> 
> Bodhi is a minimalist distro, 

The subject is not Bodhi; it is **Moksha desktop**. There are hundreds of distros but a few desktops and very few integrated desktops.

In fact, I am looking for a desktop which provides at minimum four quality (and problem-free) features, oriented for user's use:

- File manager (for daily use)
- Task/tool bars management (only once after desktop installation and occasionally after any app installation)
- Menu editor (only once after desktop installation and occasionally after any app installation)
- Hotspot management (for daily use) (Network manager is obvious that it should be included)

It is very hard to remember any other exclusively-desktop-provided feature for the user that I have used the last years. Maybe an update manager but I could live by only using synaptic.

>so there is no default application to provide any kind of menu editing beyond adding a new desktop file or editing an existing one. You are free to install such an application

Any third application is universal and may have inconsistencies. IMHO, menu editor should be built in-house for integration and quality reasons. If there is one you believe that it works fine you could include it as default or even better you could slightly modified to work fluently and consistently with Moksha desktop, if required.

>  or to directly edit the menu file itself, or create or install another menu file. See /etc/xdg/menus as well as look up the freedesktop menu specifications.

2021! In a few years from now, people will have vacations on the Moon, and you tell me to edit files...
 
> 
> I am unsure on hotspot creation. If network manager does not provide it out of the box then you most likely need to install some stuff.

Which one does KDE plasma use? With one click to open the network menu and another one to create an anonymous open-to-all connection is fantastic!

> Keep in mind any applications libraries found in Ubuntu's repository can be installed on Bodhi. And all such applications should work UNLESS they are specific to a certain Desktop Environment.

I do not know if Ubuntu provides quick access to hotspot creation but other DEs based on Gtk libraries does not which *may* mean that there are not such apps/libs. What such apps/libs do you know?

*Thanks for your time!*

**Comment by tenplus1** [comment]
Is it possible to move Moksha desktop onto using wlroots to give wayland support ?
